### DISCUSSION TITLE
Getting errors when trying to init shadcn - Need help troubleshooting

### DISCUSSION THREAD

**Discussion by IPohjola** [shadcn-ui/ui#5765] [General]
Hello! I'm trying to install and init shadcn-ui in a fresh new project with the "npx shadcn@latest init" command and I keep getting this error 
![image](https://github.com/user-attachments/assets/35703a65-645d-478d-a3b5-75417558ba2b)

I've tried both with a React project made with vite, and a NextJS project, and I get the same errors. I've tried deleting the npm cache and also deleting and reinstalling npm itself, but I keep getting this error. Any idea what could be causing it and how to fix it?

**Comment by Attilio81** [comment]
I'm happy to find this post, I've been trying to install it all afternoon but I'm getting the same problem

**Reply by IPohjola** [reply]
And I'm happy to know I'm not alone with this!

If you find a solution could you please let me know what it was? I shall do the same.
